commit 76b428e04810f8aa8f59859c0124e38367850581 (refs/changes/61/2861/2)
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2018-01-22T13:47:02-07:00 (1 year, 9 months ago)
    
    MANTA-3474 Use faucet for muskie test output formatting
    Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>

diff --git a/Makefile b/Makefile
index 9544f59..5f5f47a 100644
--- a/Makefile
+++ b/Makefile
@@ -63,8 +63,9 @@ manta-scripts: deps/manta-scripts/.git
 
 .PHONY: test
 test: $(STAMP_NODE_MODULES)
-	$(NODE) ./node_modules/.bin/nodeunit --reporter=tap \
-	    test/*.test.js test/mpu/*.test.js
+    $(NODE) ./node_modules/.bin/nodeunit --reporter=tap \
+    test/*.test.js test/mpu/*.test.js | \
+    $(NODE) ./node_modules/.bin/faucet
 
 .PHONY: release
 release: all docs
diff --git a/package.json b/package.json
index 5470605..6cad866 100644
--- a/package.json
+++ b/package.json
@@ -21,6 +21,7 @@
         "dashdash": "1.3.2",
         "deep-equal": "0.0.0",
         "dtrace-provider": "0.2.8",
+        "faucet": "0.0.1",
         "http-signature": "1.1.0",
         "jsprim": "^1.3.1",
         "kang": "1.1.0",
